.github/scripts/delete_stacks.sh

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
-ACTIVE_STACKS=$(aws cloudformation list-stacks | jq -r '.StackSummaries[] | select ( .StackStatus != "DELETE_COMPLETE" ) | select( .StackName | capture("^pr-(sandbox-)?(\\d+)$") ) | .StackName ')
+ACTIVE_STACKS=$(aws cloudformation list-stacks | jq -r '.StackSummaries[] | select ( .StackStatus != "DELETE_COMPLETE" ) | select( .StackName | capture("^pfp-pr-(sandbox-)?(\\d+)$") ) | .StackName ')
```

Comment:
sandbox capture is now incorrect as I found this morning on psu repo. See https://github.com/NHSDigital/eps-prescription-status-update-api/blob/main/.github/scripts/delete_stacks.sh

Reply:
also - we probably want to do it twice to capture old style stack names and new style ones (until we get rid of old style ones)